libs/frontend/util-site-deserialize/src/lib/deserialize-site.ts

```diff
@@ -43,6 +43,8 @@ export const deserializeEditor = (
         builderScale: serializedEditor.builderScale,
         cssPseudoClass: serializedEditor.cssPseudoClass,
         templatesShown: serializedEditor.templatesShown,
+        // Added 231201
+        componentMenuCollapses: serializedEditor.componentMenuCollapses ?? {},
```

Comment:
Instead of making new context fields optional, I've been adding a default in the deserialization function. It's not a big difference, but should make the application code slightly cleaner, and we can include all these "defaults" in a central location.